### DISCUSSION TITLE
Подсказки в поле населённый пункт/адрес в checkout

### DISCUSSION THREAD

**Discussion by joidru** [cdek-it/wordpress#73] [Ideas]
Здравствуйте, в вашем плагине случаем нет функционала подсказок при вводе населенного пункта и адреса в checkout как в сервисе dadata? Понимаю что можно сделать, но может уже есть, вы в своем виджете используете геокодер от Яндекса а у Яндекса есть еще suggest api можно было бы добавить за одно если нет такого.

**Comment by vermorag** [comment]
Добрый день. Сейчас в плагине нет функционала подсказок, поскольку для этого существуют отдельные плагины от сторонних разработчиков и других компаний.
Яндексовский саджест - это отдельный продукт и его использование необходимо реализовывать в рамках отдельной интеграции с Яндексом. Плюс это дополнительный платный продукт, не все захотят дополнительно за него платить, чтобы использовать виджет и/или плагин. Однако его использование мы рассматривали в рамках виджета и он может быть включен в будущих выпусках в рамках опциональной возможности.
